Makefile

```
@@ -1,9 +1,10 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
-.PHONY: build re-build
+.PHONY: build-image rebuild-image build-test-image rebuild-test-image
 
 IMAGE_NAME ?= tokman
+TEST_IMAGE ?= tokman.test
```

```suggestion
TEST_IMAGE ?= tokman-test
```

`.` in an image name seems weird.

---

Changed as requested.